flink-formats/flink-avro-glue-schema-registry/pom.xml

```diff
@@ -162,6 +162,13 @@ under the License.
 			<groupId>software.amazon.glue</groupId>
 			<artifactId>schema-registry-serde</artifactId>
 			<version>${glue.schema.registry.version}</version>
+			<exclusions>
+				<!-- Testcontainers dependency includes a newer version -->
+				<exclusion>
+					<groupId>org.jetbrains</groupId>
+					<artifactId>annotations</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
This change looks odd to me. Test containers should be only used during testing while you are excluding the dependency also on production side. Can you double-check that we are not braking anything?